tests/VirtoCommerce.NotificationsModule.Tests/UnitTests/NotificationSenderUnitTests.cs

```diff
@@ -130,7 +130,7 @@ public async Task InvoiceEmailNotification_SentNotification()
             //Arrange
             var language = "default";
             var subject = "Invoice for order - <strong>{{ customer_order.number }}</strong>";
-            var body = TestUtility.GetStringByPath($"Content\\{nameof(InvoiceEmailNotification)}.html");
+            var body = Path.Combine("Content", $"{nameof(InvoiceEmailNotification)}.html");
```

Comment:
TestUtility should be fixed instead of removing it, now just get the string instead of template file content.

Reply:
reverted